assets/javascripts/modules/analytics/appnexus.js

```diff
@@ -0,0 +1,45 @@
+define([
+    'modules/analytics/analyticsEnabled',
+    'modules/analytics/dntEnabled'
+], function (analyticsEnabled, dntEnabled) {
+    'use strict';
+
+    var segmentedPageCodes = {
+        'digital': {
+            'Landing': '8326260',
+            'Checkout': '8326264',
+            'Thankyou': '836984'
+        },
+        'paper-digital': {
+            'Landing': '8326267',
+            'Checkout': '8326269',
+            'Thankyou': '836986'
+        },
+        'paper': {
+            'Landing': '8326272',
+            'Checkout': '8326273',
+            'Thankyou': '836985'
+        }
+    };
+
+    function init() {
+        if (!(guardian && guardian.pageInfo && guardian.pageInfo.productData && guardian.pageInfo.productData.productSegment)) { return; }
+
+        var pageCodes = segmentedPageCodes[guardian.pageInfo.productData.productSegment];
+        if (!pageCodes) { return; }
+
+        var pageType = guardian.pageInfo.pageType;
+        if (!pageType) { return; }
+
+        var oImg = document.createElement('img');
+        oImg.setAttribute('src', 'https://secure.adnxs.com/seg?t=2&add=' + pageCodes[pageType]);
```

Comment:
The t=2 refers to how the pixel was exported from the platform. There are two options, for it to be exported as Javascript or an image. The first option adds a t=1, whereas the second a t=2.